modules/account-lib/src/coin/dot/iface.ts

```diff
@@ -170,3 +169,14 @@ export type FeeOptions = {
   formula?: 'fixed' | 'feeRate' | 'perKB' | 'custom';
   type?: 'base' | 'max' | 'tip';
 };
+
+export interface TransactionExplanation extends Omit<BaseTransactionExplanation, 'id'> {
+  type: TransactionType;
+  forceProxyType?: proxyType;
+  to?: string;
```

Comment:
This is actually a list because you can have multiple outputs. If so, we should probably put it in the `BaseTransactionExplanation` as an array of strings